consensus/service/src/byzantine_ledger/mod.rs

```diff
@@ -62,23 +62,27 @@ impl ByzantineLedger {
         peer_manager: ConnectionManager<PC>,
         ledger: L,
         tx_manager: TxManager<E, UI>,
-        broadcaster: Arc<Mutex<ThreadedBroadcaster>>,
+        broadcaster: Arc<Mutex<dyn Broadcast>>,
         msg_signer_key: Arc<Ed25519Pair>,
         tx_source_urls: Vec<String>,
         opt_scp_debug_dump_dir: Option<PathBuf>,
         logger: Logger,
     ) -> Self {
         let (sender, receiver) =
             mc_util_metered_channel::unbounded(&counters::BYZANTINE_LEDGER_MESSAGE_QUEUE_SIZE);
-        let tx_manager_validate = tx_manager.clone();
-        let tx_manager_combine = tx_manager.clone();
-        let scp_node = Node::new(
-            node_id.clone(),
-            quorum_set.clone(),
-            Arc::new(move |tx_hash| tx_manager_validate.validate_tx_by_hash(tx_hash)),
-            Arc::new(move |tx_hashes| tx_manager_combine.combine_txs_by_hash(tx_hashes)),
-            logger.clone(),
-        );
+
+        let scp_node = {
```

Comment:
I moved the clones inside of a scope. The clones are kind of a funky implementation detail, and I think this encapsulates them nicely.